src/test/kotlin/com/kafkakotlin/demo/kafka/producer/KafkaProducerTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.kafkakotlin.demo.kafka.producer
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.kafkakotlin.demo.users.User
+import com.kafkakotlin.demo.users.UserSerde
+import org.apache.kafka.clients.consumer.Consumer
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.kafka.core.ConsumerFactory
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory
+import org.springframework.kafka.test.EmbeddedKafkaBroker
+import org.springframework.kafka.test.context.EmbeddedKafka
+import org.springframework.kafka.test.utils.KafkaTestUtils
+import org.springframework.test.context.junit.jupiter.SpringExtension
+
+@SpringBootTest
+@EmbeddedKafka(partitions = 1, topics = ["user-topic"], bootstrapServersProperty = "spring.kafka.bootstrap-servers", brokerProperties = ["listeners=PLAINTEXT://localhost:9093", "port=9093"])
+@ExtendWith(SpringExtension::class)
+internal class KafkaProducerTest() {
+    private val objectMapper = jacksonObjectMapper()
+
+    @Autowired
+    private lateinit var underTest: KafkaProducer
+
+    @Autowired
+    private lateinit var embeddedKafkaBroker: EmbeddedKafkaBroker
+
+    @Test
+    fun `should put a message to the state store`() {
```

Comment:
Is this test name still appropriate? Doesn't look like its doing anything with state stores

Reply:
Ah good point 👍